docs/troubleshoot-collection.md

```diff
@@ -49,6 +52,12 @@ To set your namespace context more permanently, you can run
 kubectl config set-context $(kubectl config current-context) --namespace=sumologic
 ```
 
+## Collection dashboards
+
+Please consult this when diagnosing issues before diving into Kubernetes directly.
+
+[Zaidan Collection](https://stagdata.long.sumologic.net/ui/#/library/folder/20836981)
```

Comment:
This is not accessible

Reply:
right! I was thinking about support when I added this, but we should probably only add links here that the customer can access.

Reply:
Yes, it it's not an internal documentation